turbinia/pubsub.py

```diff
@@ -23,7 +23,7 @@
 from six.moves import xrange
 
 from google.cloud import exceptions
-from google.cloud import pubsub
+import libcloudforensics.providers.gcp.internal.common as gcp_common
```

Comment:
additional comments on unchanged lines:
- line 99 ->  this one (below) is the challenging one to use  gcp_common.ExecuteRequest and is actually better to leave as is (using google.cloud.pubsub) otherwise we have to implement the async polling logic which I believe much effort than primarily estimated and we should not go forward
  self.subscription = self.subscriber.subscribe(
        subscription_path, self._callback)
- line 139 -> future = self.publisher.publish(self.topic_path, data) should be changed to execute the https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics/publish  using the gcp_common.ExecuteRequest method